src/pages/[platform]/build-a-backend/data/data-modeling/add-fields/index.mdx

```diff
@@ -233,6 +233,9 @@ const schema = a.schema({
   }),
 }).authorization((allow) => allow.publicApiKey());
 ```
+
+If you're working with a Postgres datasource, `.default()` can be used to represent [serial fields](/[platform]/build-a-backend/data/connect-to-existing-data-sources/connect-postgres-mysql-database/#mapping-of-sql-data-types-to-field-types-for-auto-generated-schema).
```

Comment:
Do we need to specify that `.default()` can only be applied to `.integer()` fields?

Reply:
Reworded to make integer fields more explicit
> If you're working with a Postgres datasource, `.default()` can be applied to integer fields to represent serial fields.

Reply:
does `.default()` require an argument here similar to the graphql directive?

Reply:
Actually we've removed the required arg for both, and `.default()` maps to `@default`. There are checks in place so that this special (no argument) case can only be applied if:
1. Datsource is Postgres
2. Fieldtype is integer